site/AntDesign.Docs/Demos/Components/Button/doc/index.en-US.md

```diff
@@ -31,6 +31,7 @@ And 4 other properties additionally.
 
 | Property | Description | Type | Default Value | Version 
 | --- | --- | --- | --- |
+| AriaLabel | Sets the aria-label attribute         | string    | null         |
```

Comment:
It's also need to add the property in the zh-CN doc.

Reply:
Added.  For my knowledge, does someone handle the translation at a later point?

Reply:
That's fine.